climakitae/explore/agnostic.py

```diff
@@ -38,9 +38,7 @@ def create_lookup_tables():
 def _create_time_lut(gcms):
     """Prepare lookup table for converting warming levels to times."""
     # Read in simulation vs warming levels (1.5, 2, 3, 4) table
-    df = pd.read_csv(
-        "~/src/climakitae/climakitae/data/gwl_1850-1900ref_agnostic_tools.csv"
-    )
+    df = pd.read_csv("~/src/climakitae/climakitae/data/gwl_1850-1900ref.csv")

```

Comment:
Ignore this `read_csv`, I will fix this tomorrow with the correct relative path reading